.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+# Brief Description
+
+## Problem:
+
+- Describe the problem this PR solves here. 
+- If you're fixing an issue, say "Fixes #X".
+
+## Solution:
+
+- Describe the solution used to overcome the problem.
+
+## Additional Information:
+
+- Any additional information needed to better understand this PR.
+
+
+## Checklist (if applicable)
+
+- [ ] Documentation 
+- [ ] Tests
```

Comment:
@alice-i-cecile thoughts?
(should i remove this, add more boxes, etc...?)

Reply:
More boxes! Example is another good candidate.